GooglePlayInstant/Tests/Editor/QuickDeploy/HttpRequestHelperTest.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using GooglePlayInstant.Editor.QuickDeploy;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace GooglePlayInstant.Tests.Editor.QuickDeploy
+{
+    /// <summary>
+    /// Tests for HttpRequestHelper class methods.
+    /// </summary>
+    [TestFixture]
+    public class HttpRequestHelperTest
+    {
+        /*
+         * Testing strategy:
+         *     - Test helper methods that manipulate data that is fed into instantiation of WWW instances.
+         *     - Exclude methods that instantiate the WWW class from unit tests to avoid sending HTTP requests.
+         */
+
+        [Test]
+        public void TestGetWwwForm()
```

Comment:
With the use of helper methods it's very difficult for me to tell the test is correct. This is more like what I'd like to see:
```
var postParams = new Dictionary<string, string> {{"a", "b"}, {"c", "d"}};
var form = HttpRequestHelper.GetWwwForm(postParams);
Assert.AreEqual("?a=b&c=d", Encoding.UTF8.GetString(form));
```
I'd also like to see tests of boundary cases: null and empty dictionaries.

Reply:
Addressed.